src/System.Data.SqlClient/tests/FunctionalTests/System.Data.SqlClient.Tests.csproj

```diff
@@ -1,8 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
```

Comment:
Even though System.Data.SqlClient might not have any OS-specific stuff the test project here does have a configuration section for the OS-specific stuff. If it isn't OS specific then we should also clean-up the configurations.